src/software/jetson_nano/display/rotary_encoder/rotary_encoder.py

```diff
@@ -116,8 +116,10 @@ def on_rotation(channel):
 
             if self.count // self.transitions_per_rotation != 0:
                 if self.dir == CLOCKWISE:
+                    print("rotary_encoder: clockwise rotation")
```

Comment:
why are there prints?

Reply:
its useful to know if the encoder is clicking, you won't see them unless you do
`sudo systemctl status display`

its useful to have imo